java/server/src/org/openqa/selenium/grid/data/NodeStatus.java

```diff
@@ -44,7 +44,7 @@
   public NodeStatus(
       NodeId nodeId,
       URI externalUri,
-      int maxSessionCount,
+      Integer maxSessionCount,
```

Comment:
This change shouldn't be necessary for this PR. Please remove it.

Reply:
Bazel build fails otherwise. I have removed it from this PR.